code/src/main/java/com/onlinecontacttracing/messaging/SendEmails.java

```diff
@@ -0,0 +1,94 @@
+package com.onlinecontacttracing.messaging;
+
+import com.onlinecontacttracing.messaging.MessageServlet;
+import com.onlinecontacttracing.messaging.SystemMessage;
+import com.onlinecontacttracing.messaging.LocalityResource;
+import com.onlinecontacttracing.messaging.MessagingSetup;
+import javax.mail.internet.MimeMessage;
+import java.util.ArrayList;
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+// import com.google.api.client.util.FileDataStoreFactory;
+import com.google.api.services.gmail.Gmail;
+import com.google.api.services.gmail.GmailScopes;
+import com.google.api.services.gmail.model.Label;
+import com.google.api.services.gmail.model.ListLabelsResponse;
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.auth.oauth2.TokenResponse;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.GeneralSecurityException;
+import java.util.Collections;
+import java.util.List;
+
+public class SendEmails {
+
+  private String emailSubject;
+  private String emailBody;
+  private MessageServlet messageObject;
+  private ArrayList<PotentialContact> contactsList;
+  private Gmail service;
+  private PositiveUser user;
+
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  private static final String TOKENS_DIRECTORY_PATH = "tokens";
+  private static final List<String> SCOPES = Collections.singletonList(GmailScopes.GMAIL_SEND);
+  private static final String CREDENTIALS_FILE_PATH = "/credentials.json";//rename to be likek authconfigurationdata, also need to remove from github
```

Comment:
nit: typo, also move above the code, so

// rename to be like ...
private static final String ...